docs/UserGuide.md

```diff
@@ -203,7 +203,7 @@ Format: `find [n/NAME] [s/SCHOOL] [y/YEAR]`
 * The order of the optional fields do not matter. e.g `n/Hans s/River Valley` will match `s/River Valley n/Hans`
 * Only full words will be matched. e.g `han` will not match `hans`.
 * For the name, students with a name that matches any whole keyword specified for the name will be considered to match for the name.
-* For the school, students with a school that contains any keyword specified for the school will be considered to match for the school.
+* For the school, students with a school that contains all keyword specified for the school will be considered to match for the school.
 * Only students matching all criteria specified will be returned (i.e `AND` search).
```

Comment:
You might have to rewrite this whole segment, considering that this is no longer that accurate:
1. full word match only applies to name search
2. year search has not been explained

Perhaps for reference, 204-206 could be:
"For name search, only students whose names contain at least one of the given keywords will be matched. Only full words will be matched, e.g `han` will not match `hans`."
"For school search, only students whose school contains the entire given keyword will be matched, e.g `Yishun` will match `Yishun Sec` but not `Yishun Secondary`."

Reply:
Okay thanks for pointing the potential misreading in line 204! For line 206 I don't get what you mean by "e.g `Yishun` will match `Yishun Sec` but not `Yishun Secondary`." as `Yishun` currently will match `Yishun Secondary`.

Reply:
Also, I think `the entire given keyword` brings up the possibility of confusion as this can be interpreted to mean that the entire keyword string must be found in one chunk.

Reply:
Also, would `the entire given keyword` brings up the possibility of confusion as this can be interpreted to mean that the entire keyword string must be found in one chunk.

Reply:
Hmmm... perhaps it could be
"e.g `Yishun` will find all instances of `Yishun Sec` but `Yishun Secondary` will not."
and "whose school contain the entire set of given school keywords"